terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -x
+exec > >(tee /tmp/user-data.log|logger -t user-data ) 2>&1
+
+#SETTING UP HOSTNAME
+
+REGION="${region}"
+IP=`curl -s http://169.254.169.254/latest/meta-data/public-ipv4`
+echo "aws-$REGION-$IP" > /etc/hostname
+hostname "aws-$REGION-$IP"
+
+#SETUP DATADOG
+apt-get update
+apt-get install awscli -y
```

Comment:
I prefer to have to in the `package` image.

Reply:
moved